mkosi.md

```diff
@@ -734,6 +734,11 @@ details see the table below.
   in systemd-nspawn's man page:
   https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html#-n
 
+`--ephemeral`
+: When used with the shell, boot or qemu verbs, this option runs the specified
+  verb on a temporary snapshot of the output image that is removed immediately
+  when the container terminates.
```

Comment:
Maybe add here, that it uses the `--ephemeral` switch for nspawn and a copy of the image for qemu

Reply:
I wouldn't go into specifics, I can copy paste some more info from the nspawn man page though

Reply:
Let's not add too many implementation details. I extended with some more info from nspawn's man page that informs the user that the copy is more efficient on filesystems that support reflink.